Alec thermal setup window

RailgunZx opened the pull request:
## Describe your changes
The layout of the Thermal Setup window is mostly complete, however I could not figure out how to add  components to the page for the purpose of adding or removing devices from the table inside of the window.
## Issue link and documentation link
[📚Documentation](https://link-url-here.org)
[💻Issue](https://link-url-here.org)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] I have added the necessary documentation.

RailgunZx committed:
Create ThermalSetupWIndow.tsx

RailgunZx committed:
work in progress of the window

RailgunZx committed:
more progress

RailgunZx committed:
more progress

RailgunZx committed:
even more progress

RailgunZx committed:
progress

RailgunZx committed:
Finished window layout

The window is mostly complete, but I was unable to add the functionality of "Add Device" and "Remove Device" for the sake of the layout. That portion remains incomplete

kevin-german commented on src/components/optimizationSetup/ThermalSetupWIndow.tsx:
```diff
@@ -0,0 +1,108 @@
+    
+    return (
+        <div className="mainDiv">
+            <div className="thermalSetupText">Thermal Setup</div>
```
The title here for 'Thermal Setup' isn't needed and can be removed, since the popup that this component is being called in already has the title.

kevin-german commented on src/components/optimizationSetup/ThermalSetupWIndow.tsx:
I'd take a look at revising the way the sizing of the component works, in terms of it being dynamically resized. On smaller screens, the component extends outside of the popup that it is being called in; this would apply to both width and height.
![Screen Shot 2023-04-24 at 11 41 43 AM](https://user-images.githubusercontent.com/98424956/234062270-42080903-f991-42dc-a484-c98987f27ce8.png)

kevin-german committed:
Thermal Setup Component in to Popup

Added the thermal setup component to the OptimizationSetup.tsx file so it appears in the popup.

kevin-german committed:
Update OptimizationSetup.tsx

Removed an unnecessary p tag.

restyled-io[bot] committed:
Restyled by whitespace (#63)

Co-authored-by: Restyled.io <commits@restyled.io>

RailgunZx committed:
fixed window scaling

restyled-io[bot] committed:
Restyled by whitespace (#70)

Co-authored-by: Restyled.io <commits@restyled.io>

kevin-german committed:
Merge branch 'main' into alec-thermal-setup-window

kevin-german reviewed (dismissed):
Looks good to me.

restyled-io[bot] committed:
Restyled by jq (#71)

Co-authored-by: Restyled.io <commits@restyled.io>
